doc/snippets/Microsoft.Data.SqlClient/SqlConnectionStringBuilder.xml

```diff
@@ -160,10 +183,21 @@ Modified: Data Source=(local);Initial Catalog=AdventureWorks;Integrated Security
             <related type="Article" href="https://msdn.microsoft.com/library/745c5f95-2f02-4674-b378-6d51a7ec2490">Working with Connection Strings</related>
             <related type="Article" href="/sql/connect/ado-net/overview-sqlclient-driver">Overview of the SqlClient driver</related>
         </AttachDBFilename>
+        <AttestationProtocol>
+            <summary>Gets or sets the value of Attestation Protocol.</summary>
+            <returns>Returns the attestation protocol.</returns>
+        </AttestationProtocol>
         <Authentication>
-            <summary>Gets the authentication of the connection string.</summary>
-            <value>The authentication of the connection string.</value>
-            <remarks>To be added.</remarks>
+            <summary>Gets or sets the authentication method used for [Connecting to SQL Database By Using Azure Active Directory Authentication](https://azure.microsoft.com/documentation/articles/sql-database-aad-authentication/#7-connect-to-your-database-by-using-azure-active-directory-identities).</summary>
+            <value>The authentication method of the connection string.</value>
+            <remarks>
+            <format type="text/markdown"><![CDATA[  
+  
+## Remarks  
+Currently Active Directory Integrated and Active Directory Interactive modes of authentication are supported only for .NET Framework.
```

Comment:
Please update this (in all spots) according to new support in MDS. More info here: https://docs.microsoft.com/en-us/sql/connect/ado-net/sql/azure-active-directory-authentication?view=sql-server-ver15#setting-azure-active-directory-authentication (you may redirect to docs too)